dspback/schemas/zenodo/schema.json

```diff
@@ -324,20 +324,14 @@
     "references": {
```

Comment:
According to Zenodo API documentation, the field 'references' is an array of strings: https://developers.zenodo.org/#representation

I've adjusted our schema accordingly.